eventhubs/client/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubConsumer.java

```diff
@@ -20,27 +21,37 @@
 import static com.azure.messaging.eventhubs.EventHubConsumerOptions.MINIMUM_PREFETCH_COUNT;
 
 /**
- * This is a logical representation of receiving from an Event Hub partition.
- *
- * <p>
- * A {@link EventHubConsumer#receive()} is tied to a Event Hub partitionId + consumer group combination.
+ * A consumer responsible for reading {@link EventData} from a specific Event Hub partition and as a member of a
+ * specific consumer group.
  *
  * <ul>
  * <li>If {@link EventHubConsumer} is created where {@link EventHubConsumerOptions#ownerLevel()} has a
- * value, then Event Hubs service will guarantee only one active receiver exists per partitionId and consumer group
- * combination. This is the recommended approach to create a {@link EventHubConsumer}.</li>
+ * value, then Event Hubs service will guarantee only one active consumer exists per partitionId and consumer group
+ * combination. This consumer is sometimes referred to as an "Epoch Consumer."</li>
  * <li>Multiple consumers per partitionId and consumer group combination can be created by not setting
- * {@link EventHubConsumerOptions#ownerLevel()} when creating receivers.</li>
+ * {@link EventHubConsumerOptions#ownerLevel()} when creating consumers. A maximum of five concurrent consumers are
```

Comment:
Re:  "A maximum of five... "    

I'd be very careful about stating limits in docstrings;  these are purview of the service and could be changed at any time without the client having awareness.

Reply:
Good point, I'll remove that.